src/medCore/gui/toolboxes/medToolBox.cpp

```diff
@@ -382,3 +389,15 @@ void medToolBox::addToolBoxConnections(medJobItem* job)
     connect (job, SIGNAL (failure   (QObject*)),    this, SLOT   (setToolBoxOnReadyToUse()));
     connect (job, SIGNAL (activate(QObject*,bool)), getProgressionStack(), SLOT(setActive(QObject*,bool)));
 }
+
+void medToolBox::enableOnProcessSuccessImportOutput(medJobItem* job, bool enable)
+{
+    if (enable)
+    {
+        connect(job, SIGNAL(success(QObject*)), this->getWorkspace(), SLOT(importProcessOutput()));
```

Comment:
For safety this should be a unique connection to avoid duplicate connects if this function is called multiple times.